django/db/backends/sqlite3/operations.py

```diff
@@ -314,6 +314,8 @@ def converter(value, expression, connection):
         else:
             def converter(value, expression, connection):
                 if value is not None:
+                    if isinstance(value, str):
+                        return create_decimal(float(value))
```

Comment:
I think we should make `Value` inherit from `SQLiteNumericMixin`.

Reply:
I just patched it so I could get my testDB code working, don't really know enough about the inner workings of Django to do a proper patch. If someone else wants to do it properly then go for it.

Reply:
OK, thanks. Don't worry someone will prepare a patch.